internal/auth/auth.go

```diff
@@ -40,38 +40,41 @@ const (
 
 // The following constants define possible operation actions.
 const (
-	ActionRead               = "read"
-	ActionVerify             = "verify"
-	ActionDischargeFor       = "dischargeFor"
-	ActionDischarge          = "discharge"
-	ActionCreateAgent        = "createAgent"
-	ActionCreateParentAgent  = "createParentAgent"
-	ActionReadAdmin          = "readAdmin"
-	ActionWriteAdmin         = "writeAdmin"
-	ActionReadGroups         = "readGroups"
-	ActionWriteGroups        = "writeGroups"
-	ActionReadSSHKeys        = "readSSHKeys"
-	ActionWriteSSHKeys       = "writeSSHKeys"
-	ActionLogin              = "login"
-	ActionReadDischargeToken = "read-discharge-token"
+	ActionRead                     = "read"
+	ActionVerify                   = "verify"
+	ActionDischargeFor             = "dischargeFor"
+	ActionDischarge                = "discharge"
+	ActionCreateAgent              = "createAgent"
+	ActionCreateParentAgent        = "createParentAgent"
+	ActionReadAdmin                = "readAdmin"
+	ActionWriteAdmin               = "writeAdmin"
+	ActionReadGroups               = "readGroups"
+	ActionWriteGroups              = "writeGroups"
+	ActionReadSSHKeys              = "readSSHKeys"
+	ActionWriteSSHKeys             = "writeSSHKeys"
+	ActionLogin                    = "login"
+	ActionReadDischargeToken       = "read-discharge-token"
+	ActionRemoveUserMFACredentials = "remove-user-mfa-credentials"
```

Comment:
I know "read-discharge-token" is formatted like this, but I'd prefer if we had a more consistent format. I think "removeUserMFACredentials"

Reply:
done